websites/N/Nexon Logistics/presence.ts

```diff
@@ -53,9 +53,9 @@ presence.on("UpdateData", async () => {
 					];
 					break;
 				case document.location.pathname.includes("/index"):
-					presenceData.details = "Browsing Nexon Homepage";
+					presenceData.details = "Viewing Nexon Homepage";
 					presenceData.state =
-						document.querySelectorAll<HTMLHeadingElement>("h1")[1].textContent;
```

Comment:
In this case, this may have an unintended effect, as you were using selecting the second `h1` element (`[1]`)

Reply:
I intend to use the second h1 element.

Reply:
Exactly, in your latest commit, you may have accidentally replaced this to use the first h1 element.